src/Blockfrost.Api/Blockfrost.Api.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>net5.0;netstandard2.1;net472</TargetFrameworks>
```

Comment:
why do we need to support .NET5, .NET Standard and .NET Framework? Shouldn't .NET Standard be compatible with both .NET5 and .NET Framework?

Reply:
Ah never mind, .NET Standard doesn't support .NET Framework.
https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support

Reply:
Nuget used to have some funny behviour if you didnt add net472 explicitly... and yes we can remove net5